packages/mobile/README.md

```diff
@@ -526,6 +527,24 @@ We try to minimise the differences between running Valora in different modes and
 - Valora uses Crowdin Over-The-Air (OTA) content delivery to enable dynamic translation updates. The OTA translations are cached and used on subsequent app loads instead of the strings in the translation files of the app bundle. This means that during development, the app will not respond to manual changes of the translation.json files.
 - In development mode, analytics are disabled.
 
+### Vulnerabilities found in dependencies
+
+We have a script to [check for vulnerabilities](scripts/ci_check_vulnerabilities.sh) in our dependencies.
+
+In case vulnerabilities are reported, check to see if they apply to production and if they have fixes available.
+
+If they apply to production, start a discussion in our [#on-call](https://valora-app.slack.com/archives/C02N3AR2P2S) channel.
+
+Then if they have fixes available, update the dependencies using [Renovate](https://github.com/valora-inc/wallet/issues/1716) or manually:
+
+- If it's a direct dependency, update the dependency in `package.json`.
+- If it's a transitive dependency, you can manually remove the transitive dependency in `yarn.lock` and re-run `yarn install` to see if it can use the fixed version. If the sub dependency is pinned somewhere, you'll need to use a [yarn resolution](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/) in `package.json` to get the fixed version. Be careful with this as it can break other dependencies depending on a specific version.
+
+If they do not have fixes and they do not apply to production, you may ignore them:
```

Comment:
should this be an "or"? If there are no fixes and it applies to production, what then?

Reply:
> If there are no fixes and it applies to production, what then?

Good question, it would fall into the #on-call case, stated earlier.
Not how to present this more clearly.